service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/http.proxy.use.forwarded.headers.rpt

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect http://localhost:8110/
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+
+write header "User-Agent" "curl/7.37.1"
+write header host
+write header "Accept" "*/*"
+write close
+
+read status "200" "OK"
+read version "HTTP/1.1"
+read header "Content-Type" "text/html"
+read header "Content-Length" "13"
+read "Hello world\r\n"
+read closed
+
+close
+closed
+
+
+accept http://localhost:8080/
+accepted
+connected
+
+read method "GET"
+read header "Via" "1.1 kaazing"
+read header "User-Agent" "curl/7.37.1"
+read header "Host" "localhost:8080"
+read header "Forwarded" "for=127.0.0.1;by=127.0.0.1;proto=http;host=localhost;port=8110"
```

Comment:
I don't see RFC specifying any "port" parameter. host parameter itself includes it.